hal/tests/TESTS/mbed_hal/stack_size_unification/CMakeLists.txt

```
@@ -3,9 +3,40 @@
 
 include(mbed_greentea)
 
+if(DEFINED TARGET_RENESAS)
```

Preexisting:
```suggestion
if ("CORTEX_A" IN_LIST MBED_TARGET_LABELS)
```
We'd better change it in main.cpp too:
```
#ifdef TARGET_CORTEX_A
```